setup.py

```diff
@@ -24,16 +24,16 @@
     history = history_file.read()
 
 requirements = [
-    "appdirs>=1.4.3",
-    "Click>=7.0",
-    "diskcache>=4.0.0",
-    "inflection>=0.3.1",
-    "iso8601>=0.1.12",
-    "Jinja2>=2.10.1,<3.0",
-    "jsonschema>=3.0.2",
-    "pep487==1.0.1",
-    "PyYAML>=3.13",
-    "yamllint>=1.18.0",
+    "appdirs==1.4.3",
```

Comment:
Can we have the change to stricter versions in a separate commit?

Reply:
Darn.  Unintentional.